### PR TITLE
Sketch index page

### PR DESCRIPTION
**Pull request by oscarjosecostero**
ref #3 

### Todo

- [x] Installed bourbon
- [x] Installed Slim
- [x] Installed Bourbon
- [x] Index page basically sketched

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,11 +4,16 @@
     <title>Easyrent</title>
```
Comment (OscarlynCostero) [inline app/views/layouts/application.html.erb]: Convert this file to slim.

File: .gitignore [hunk 2]
```diff
@@ -22,3 +22,4 @@
 
 .byebug_history
 .rubocop.yml
+.notes.txt
```
Comment (OscarlynCostero) [inline .gitignore]: Please remove this line and add this.
```
/public
/vendor
Gemfile.lock
```

**Review by OscarlynCostero — CHANGES_REQUESTED** [review]
All good! There are a few thing to change but we can change most of it on the run. 

However, I left you two comments.

**Review by gbrlmrllo — APPROVED** [review]
Good job 👍 , I made some minor changes, so I could merge.

**Pull request merged**